cirq/contrib/kraus/noisyEvalultion-cirq.py

```diff
@@ -0,0 +1,398 @@
+# This is a version of the N = 2 qubit noisty circuit evolution. The purpose is to introduce
+# into Cirq density matrix evoluton with Kraus operators. 
+# Implementation by Hrant Gharibyan, Quantum AI, Google Inc. 
+# The code is build to serve as a template for appripriate edits in Cirq 
+# to support noisy evolution. 
+
+import numpy as np
+import scipy.linalg as lg 
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_pdf import PdfPages
+from scipy.optimize import curve_fit
+from scipy.optimize import fsolve
+from scipy.optimize import brentq
+from scipy.optimize import newton_krylov
+
+
+
+
+# Useful gates we will use in the circuit simulations 
+# Single qubit gates 
+X = np.array([[0, 1], [1, 0]]); 
```

Comment:
You should use cirq's built-in gates instead of defining your own. E.g. you can use `cirq.unitary(cirq.X)`